client/defines.h

```diff
@@ -79,6 +79,7 @@ typedef enum
 #define TDNF_CONF_KEY_PLUGIN_PATH         "pluginpath"
 #define TDNF_CONF_KEY_PLUGIN_CONF_PATH    "pluginconfpath"
 #define TDNF_PLUGIN_CONF_KEY_ENABLED      "enabled"
+#define TDNF_CONF_KEY_EXCLUDE             "exclude"
```

Comment:
Should this be `excludepkgs`? I see `excludepkgs` but not `exclude` in https://dnf.readthedocs.io/en/latest/conf_ref.html .

Reply:
This was passed down to me in the JIRA, I'll change this to match dnf. I'll also add a new flag `--disableexcludes` to exclude those from config file as this looks useful keep around.

Reply:
This is now fixed.